Can't compile without tools

There's an error when compiling the current HEAD without tools enabled. It was introduced in 2db78e8f.

Error message:
```
scene/main/scene_main_loop.cpp:624:24: error: use of undeclared identifier 'edited_scene_root'
        return editor_hint && edited_scene_root && edited_scene_root->is_a_parent_of(p_node);
                              ^
```
 
`edited_scene_root` is only there if `TOOLS_ENABLED` is defined

--- a/scene/main/scene_main_loop.cpp
+++ b/scene/main/scene_main_loop.cpp
@@ -621,7 +621,11 @@ void SceneTree::set_editor_hint(bool p_enabled) {
 }
 
 bool SceneTree::is_node_being_edited(const Node* p_node) const {
+#ifdef TOOLS_ENABLED
 	return editor_hint && edited_scene_root && edited_scene_root->is_a_parent_of(p_node);
+#else
+	return false;
+#endif
 }
 
 bool SceneTree::is_editor_hint() const {

